Scripts/CreateProjectGlossaryMarkdown.js

```diff
@@ -0,0 +1,132 @@
+/**
+	{
+		"api":1,
+		"name":"Create Project Glossary Markdown File",
+		"description":"Type 'help' and run this script for instructions.",
+		"author":"Terry L. Lewis",
+		"icon":"colosseum",
```

Comment:
Feel free to replace with a more appropriate icon. I didn't see anything that screamed "glossary" to me....